### PR TITLE
standardjs

### PR DESCRIPTION
**Pull request by pukkamustard**
Reformat to standardjs

### CODE REVIEW

**Comment by pukkamustard** [comment]
I'm starting to really like standardjs...caught a few bugs just by reformating.

@knuton Can you check this in particular: https://github.com/dividat/driver/pull/24/files#diff-1c90ff38a08209f9ebd4d05d1e43358eR11

**Comment by knuton** [comment]
Yes, I think the linter is right. The use of `return` was suggested in the electron-winstaller Readme or some such place, without further explanation. The fact that it works is due to an implementation detail of Node.

Using `process.exit` should be fine.

File: src/main.js [hunk 1]
```diff
@@ -1,73 +1,78 @@
...
+  // TODO: check if it makes sense to use process.exit() here. `return` causes a standardjs syntax error, as it can only be used in a function
+  process.exit()
+  // return
+}
 
-const Config = require('electron-config');
-const config = new Config();
-const constants = require('./constants');
+// const Config = require('electron-config')
+// const config = new Config()
```
Comment (knuton) [inline src/main.js]: Why is this commented out?

File: src/main.js [hunk 2]
```diff
@@ -1,73 +1,78 @@
...
+const Tray = electron.Tray
+
+const path = require('path')
 
 // Run handler for Windows installer options
-if (require('./installation/handle-squirrel-events')()) return;
+if (require('./installation/handle-squirrel-events')()) {
+  // TODO: check if it makes sense to use process.exit() here. `return` causes a standardjs syntax error, as it can only be used in a function
+  process.exit()
+  // return
```
Comment (knuton) [inline src/main.js]: Can be removed.

File: src/main.js (hunk 1 above)
Comment (pukkamustard) [inline src/main.js]: Was commented out because unused in main.js. I removed it.

**Pull request merged**